docs/internal_api/index.rst

```diff
@@ -123,6 +123,16 @@ Connectivity
    grid.connectivity._build_edge_node_connectivity
    grid.connectivity._build_face_edges_connectivity
 
+Geometry
+--------
+.. autosummary::
+   :toctree: _autosummary
+
+   grid.geometry._build_polygon_shells
+   grid.geometry._build_corrected_polygon_shells
+   grid.geometry._build_antimeridian_face_indices
+   grid.geometry._grid_to_polygon_geodataframe
```

Comment:
How about alphabetically ordering all of the functions under each heading/section throughout the document?

Reply:
Will tackle this in a separate PR, issue has been created for it (#407)